### DISCUSSION TITLE
Supervisor and rsyslog may not be needed

### DISCUSSION THREAD

**Discussion by jaylinski** [juanluisbaptiste/docker-postfix#54] [Ideas]
Hi!

I stumbled upon this repository because of issue #45 (which helped me a lot) and noticed that you are still using supervisord and rsyslogd.

I'm also running postfix in an alpine docker-container and was able to get rid of both supervisord and rsyslogd:

* Since version [3.3.0 (February 21, 2018)](http://www.postfix.org/announcements/postfix-3.3.0.html) it is possible to run Postfix in foreground mode via `postfix start-fg`, which eliminates the supervisord dependency.
* Since version [3.4.0 (February 27, 2019)](http://www.postfix.org/announcements/postfix-3.4.0.html) it is possible to directly log to `stdout`, which eliminates the rsyslogd dependency:   
`postconf -e "maillog_file=/dev/stdout"`.

Maybe you can simplify this image with this info.

(I read about this in detail here: https://xc2.wb1.xyz/post/how-to-run-a-postfix-mail-server-in-a-docker-container/)

**Comment by juanluisbaptiste** [comment]
Hi @jaylinski , thank you for the information, I will take a look as soon as I have some time.

**Comment by juanluisbaptiste** [comment]
@jaylinski On develop branch I have implemented this, there is juanluisbaptiste/postfix:1.1.0-develop.1 if you want to test it out before merging with pull request #59 .

**Comment by jaylinski** [comment]
Nice! I currently have no time to try this, but the change looks good. 👍

**Comment by juanluisbaptiste** [comment]
PR merged.
